models/Expense.js

```diff
@@ -0,0 +1,19 @@
+import { Decimal128, Int32 } from "bson";
```

Comment:
Are you going to need this in the future? It's not being used at the moment from what I can tell.

Reply:
Nope, I tried to use it but it didnt work so i sticked with "number" and forgot to delete the import. Thanks